rust/tests/test.pester.ps1

```diff
@@ -0,0 +1,14 @@
+param (
+    [Parameter()]
+    [string]$PackageIdentifier = $(throw "Fully qualified package identifier must be given as a parameter.")
+)
+
+$PackageVersion = $PackageIdentifier.split('/')[2]
+Describe "core/rust" {
+    Context "rustc" {
+        $OutputVariable = (hab pkg exec $PackageIdentifier rustc --version | Out-String).split(' ')[1]
```

Comment:
I think this line should be inside the `It` so that any failure in running rustc happens within the context of the test.

Reply:
Amended.